sdks/go/pkg/beam/io/rtrackers/offsetrange/offsetrange.go

```diff
@@ -208,9 +210,11 @@ func (tracker *Tracker) GetProgress() (done, remaining float64) {
 	return
 }
 
-// IsDone returns true if the most recent claimed element is past the end of the restriction.
+// IsDone returns true if the most recent claimed element is past the end of the restriction
+// or if the restriction represents no work to be done (aka the start of the restriction is
+// greater than or equal to the end).
 func (tracker *Tracker) IsDone() bool {
-	return tracker.err == nil && tracker.claimed >= tracker.rest.End
+	return tracker.err == nil && (tracker.claimed >= tracker.rest.End || tracker.rest.Start >= tracker.rest.End)
```

Comment:
Wouldn't the "done" case with claimed be `tracker.claimed >= (tracker.rest.End - 1)` since the range is [start, end)?